content-test/selectors/selectors.test.js

```diff
@@ -252,6 +254,57 @@ describe("selectors", () => {
       assert.equal(state.label, "foo.com");
     });
   });
+
+  describe("selectSitePreview", () => {
+    const siteWithMedia = {
+      url: "https://www.youtube.com/watch?v=lDv68xYHFXM",
+      images: [{url: "foo.jpg", height: IMG_HEIGHT, width: IMG_WIDTH}],
+      media: {
+        type: "video"
+      },
+    };
+    const embedPreviewURL = "https://www.youtube.com/embed/lDv68xYHFXM?autoplay=1";
+    let state;
+    beforeEach(() => {
+      state = selectSitePreview(siteWithMedia);
+    });
+
+    it("should not throw", () => {
+      assert.doesNotThrow(() => {
+        selectSitePreview({});
+      });
+    });
+
+    it("should have a preview url", () => {
+      assert.equal(state.previewURL, embedPreviewURL);
+    });
+    it("should have a thumbnail", () => {
+      assert.property(state, "thumbnail");
+      assert.isObject(state.thumbnail);
+      assert.property(state.thumbnail, "url");
+      assert.equal(state.thumbnail.url, siteWithMedia.images[0].url);
+    });
+    it("should have a type", () => {
+      assert.equal(state.type, siteWithMedia.media.type);
+    });
+    it("should return null thumbnail if no images exists", () => {
+      state = selectSitePreview(Object.assign({}, siteWithMedia, {images: []}));
+      assert.equal(state.thumbnail, null);
```

Comment:
this is fine, there's actually an `assert.isNull` method you can use if you want